sa-gateway/k8s/eureka-server-deployment.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  annotations:
+    kompose.cmd: kompose convert -f docker-compose.yml
+    kompose.version: 1.19.0 ()
+  creationTimestamp: null
+  labels:
+    io.kompose.service: eureka-server
```

Comment:
You don't need Eureka in K8S

Reply:
I agree, I'll remove it in a separate pull request